_posts/2017-02-02-Rust-1.15.md

```diff
@@ -0,0 +1,273 @@
+[`std::char::encode_utf16`]: https://doc.rust-lang.org/std/primitive.char.html#method.encode_utf16
+[`std::cell::Ref::clone`]: https://doc.rust-lang.org/std/cell/struct.Ref.html#method.clone
+[`std::io::Take::into_inner`]: https://doc.rust-lang.org/std/io/struct.Take.html#method.into_inner
+
+See the [detailed release notes][notes] for more.
+
+#### Cargo features
+
+Cargo has received an ergonomics improvement: If you have a file named
+`build.rs` in the root directory, it will [now assume that it's a build script
+][cargo/3361]. Previously, all build scripts required configuration, but this
+convention was strong within the community, so we've encoded it into Cargo.
+
+[cargo/3361]: https://github.com/rust-lang/cargo/pull/3361
+
+In this release, [Cargo build scripts no longer have access to the `OUT_DIR`
+environment variable at build time via `env!("OUT_DIR")`][cargo/3368]. They
+should instead check the variable at runtime with `std::env`. That the value
+was set at build time was a bug, and incorrect when cross-compiling. Please
+check what your packages are doing and update to use `std::env`!
+
+[cargo/3368]:  https://github.com/rust-lang/cargo/pull/3368
+
+The `cargo test` command has [gained support for a `--all` flag][cargo/3221],
+which is useful when you have a workspace.
+
+[cargo/3321]: https://github.com/rust-lang/cargo/pull/3321
+
+We now [Compile statically against the MSVC CRT][cargo/3363] on Windows, and
+[Link OpenSSL statically][cargo/3311] on Mac OS X.
+
+[cargo/3363]: https://github.com/rust-lang/cargo/pull/3363
+[cargo/3311]: https://github.com/rust-lang/cargo/pull/3311
+
+See the [detailed release notes][notes] for more.
+
+### Contributors to 1.15.0
+
+In this part of the release announcements, we usually post a list of
+contributors. However, we've recently started a new initiative, "Thanks!", to
+do this in a more comprehensive way. One issue with this section is that it
+only counted contributions to the `rust-lang/rust` repository; those who
+committed to Cargo weren't thanked, for example. We also had to manually
+generate this list, which wasn't terrible, but running the correct git commands
+to determine who contributed is exactly what code is good for!
+
+As such, you can now visit https://thanks.rust-lang.org/ to see more
+comprehensive contribution calculations. If you prefer, we also have an alias
+at https://❤.rust-lang.org as well. For now, this will only show what we've
+shown in previous release posts. We do have one additional feature, which is an
+all-time contributions list, sorted by commit count. That's located here:
+https://thanks.rust-lang.org/rust/all-time
+
+We have done some of the needed backend work to enable more repositories than
+only `rust-lang/rust`, but it's not quite done yet. If you'd like to get
+involved, please [check out thanks on
+GitHub](https://github.com/rust-lang-nursery/thanks)!
+
+We had X individuals contribute to Rust 1.15.
```

Comment:
yes, I am waiting on @brson to tag the release. There's one more below too. But thanks for pointing it ou!